channel_test.go

```diff
@@ -0,0 +1,51 @@
+package cli
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestCreateChannel(t *testing.T) {
+	app := initApp()
+	ch := initChannel(t, app)
+	t.Cleanup(func() {
+		app.Run([]string{"", "channel", "delete", "-t", "messaging", "-n", ch, "--hard"})
+	})
+}
+
+func TestGetChannel(t *testing.T) {
+	app := initApp()
+	ch := initChannel(t, app)
+	t.Cleanup(func() {
+		app.Run([]string{"", "channel", "delete", "-t", "messaging", "-n", ch, "--hard"})
+	})
+
+	err := app.Run([]string{"", "channel", "get", "-t", "messaging", "-n", ch})
+	require.NoError(t, err)
+}
+
+func TestDeleteChannel(t *testing.T) {
+	app := initApp()
+	ch := initChannel(t, app)
+	err := app.Run([]string{"", "channel", "delete", "-t", "messaging", "-n", ch, "--hard"})
+	require.NoError(t, err)
```

Comment:
Let's check that the channel is correctly deleted? Also can we add more tests for bad inputs?